gemini/function-calling/function_calling_data_structures.ipynb

```diff
@@ -0,0 +1,745 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        generation_config={"temperature": 0},
</pre><p>Can you use the GenerationConfig class?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/507/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Thanks! Fixed this and set the config globally at model instantiation time rather than individual model requests.